interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpressionParseError.scala

```diff
@@ -179,6 +179,28 @@ object SpelExpressionParseError {
     case class BadOperatorConstructionError(operator: String) extends OperatorError {
       override def message: String = s"Bad '$operator' operator construction"
     }
+
+    case class DivisionByZeroError private(expression: String) extends OperatorError {
+      override def message: String = s"Division by zero: $expression"
+    }
+
+    object DivisionByZeroError {
+      def apply(expression: String): DivisionByZeroError =
+        new DivisionByZeroError(stripOperatorAST(expression))
+    }
+
+    case class TakingModuloZeroError private(expression: String) extends OperatorError {
```

Comment:
Why "taking"? Is it commonly used with modulo? Shouldn't be just ComputingModuloZeroError? Or just ModuloZeroError?